README.md

```diff
@@ -0,0 +1,50 @@
+# jupyterlab_allofus
+
+JupyterLab extension for All of Us Jupyter notebooks.
+
+Ruby targets in libproject/devstart.rb provide targets for building and running the code, using
+Docker.
+
+## Building code
+
+To generate client libraries for talking to the workbench API based on a tagged client_api.yaml
+file referenced in package.json, run:
+
+```
+./project.rb swagger-regen
+```
+
+To build the (generated and non-generated) Typescript code, and put the output Javascript in lib, run:
+
+```
+./project.rb build
+```
+
+
+To clean the build output, run:
+
+```
+./project.rb clean
+```
+
+To install the built code in Jupyterlab, run:
+```
+./project.rb install
+```
+
+(TODO: combine build and install? Have a dev-up command that does everything?)
```

Comment:
This would definitely be better. Do you need to re-install after making code changes?